staging/src/k8s.io/component-base/metrics/prometheus/meta/metrics.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package meta
+
+import (
+	k8smetrics "k8s.io/component-base/metrics"
+)
+
+var (
+	RegisteredMetrics = k8smetrics.NewCounterVec(
+		&k8smetrics.CounterOpts{
+			Name:           "registered_metric_total",
```

Comment:
nit: should this (and others) be plural?  `registered_metrics_total`. Seems to follow from https://prometheus.io/docs/practices/naming/#metric-names

Reply:
done.